WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -138,6 +139,16 @@
 
     private static final int INTENT_COMMENT_EDITOR = 1010;
 
+    public static final String COMMENT_FILE_KEY = "commentStateFileKey";
+
+    public static final String COMMENT_STATE_KEY = "commentStateKey";
+    public static final String COMMENT_STATE_INITIAL = "commentStateInitial";
```

Comment:
If you expect more statuses in the future, I think it would make sense to have them as enum or similar. If not, do you think boolean would work here?

Reply:
Hey @khaykov I prefer an enum, so I'll update it 😄